core/uemis-downloader.c

```diff
@@ -1161,7 +1202,7 @@ static bool get_matching_dive(int idx, char *newmax, int *uemis_mem_status, devi
 	while (!found) {
 		if (import_thread_cancelled)
 			break;
-		snprintf(dive_to_read_buf, sizeof(dive_to_read_buf), "%d", dive_to_read);
+        	snprintf(dive_to_read_buf, sizeof(dive_to_read_buf), "%d", dive_to_read+1);
```

Comment:
this line starts with spaces instead of a tab. could you please use tabs for the indentation.
also, please modify `dive_to_read+1` to `dive_to_read + 1`.